src/deluge/gui/views/view.cpp

```diff
@@ -1003,7 +1005,30 @@ void View::setKnobIndicatorLevel(uint8_t whichModEncoder) {
 		    modelStackWithParam->modControllable->getKnobPosForNonExistentParam(whichModEncoder, modelStackWithParam);
 	}
 
-	indicator_leds::setKnobIndicatorLevel(whichModEncoder, knobPos + 64);
```

Comment:
It would make much more sense to quantize the values going into the stutter setting, rather than accepting continuous values and quantizing them in separate parts of the codebase.